TESTING.md

```diff
@@ -54,16 +54,50 @@ property or `MANTA_IT_NO_CLEANUP` environment variable.
 
 # Running Tests
 
-Run `mvn verify` from the project root to run all tests. Some Maven goals will
-not work
+Run `mvn verify` from the project root to run all tests. Some Maven goals will not work.
 
 While the Java Cryptography Extensions are expected to be installed, it is
-possible torun a subset of the test suite by adding
-`-DexcludedGroups=unlimited-crypto`, e.g.:
+possible to run a subset of the test suite by adding
+`-DexcludedGroups=unlimited-crypto` or `-Dgroups=headers,error`, e.g.
 ```
 mvn test -DexcludedGroups=unlimited-crypto
+mvn verify -Dgroups=headers,error
 ```
 
+Also, if we are running a defined TestNG group, invoke
+`-Dgroups=headers`, e.g:
+
+```
+mvn clean verify -Dgroups=headers
+```
+
+Note: Since we have transitioned from wildcard to explicit test definition, it is **essential**
```

Comment:
The tests will run nevertheless of being explicitly enumerated **but** they will not run if we source the `testing-it.xml` file since we have transitioned from a `package-level` -> `class-level` hierarchy. I will make sure to add this to `README` and incorporate the edit